### PR TITLE
GH-90744: Fix erroneous doc links in the sys module

### PR DESCRIPTION
**Pull request by furkanonder**
* Issue: gh-90744

### CODE REVIEW

**Comment by hugovk** [comment]
This removes the bad links, but they're no longer formatted as code:

<table>
<tr>
	<th>Before
	<th>After
<tr>
	<td>
image
	<td>
image
<tr>
    <td><a href="https://docs.python.org/3.12/library/sys.html#sys.float_info">https://docs.python.org/3.12/library/sys.html#sys.float_info</a>
    <td><a href="https://deploy-preview-101319--python-cpython-preview.netlify.app/library/sys.html#sys.float_info">https://deploy-preview-101319--python-cpython-preview.netlify.app/library/sys.html#sys.float_info</a>
</table>


And when running Sphinx in nit-picky mode (re: https://github.com/python/cpython/issues/101100), we're getting warnings about all of those:

```
Doc/library/sys.rst:14: WARNING: py:const reference target not found: epsilon
Doc/library/sys.rst:19: WARNING: py:const reference target not found: dig
Doc/library/sys.rst:22: WARNING: py:const reference target not found: mant_dig
Doc/library/sys.rst:27: WARNING: py:const reference target not found: max_exp
Doc/library/sys.rst:30: WARNING: py:const reference target not found: max_10_exp
Doc/library/sys.rst:39: WARNING: py:const reference target not found: min_exp
Doc/library/sys.rst:42: WARNING: py:const reference target not found: min_10_exp
Doc/library/sys.rst:45: WARNING: py:const reference target not found: radix
Doc/library/sys.rst:47: WARNING: py:const reference target not found: rounds
```

So shall we change all of them like this?

```diff
-    :const:`max`
+    ``max``
```

And we could also do the macros too, although that will also require adjusting the column width:

```diff
-    DBL_MAX
+    ``DBL_MAX``
```

**Comment by furkanonder** [comment]
> This removes the bad links, but they're no longer formatted as code:
> 
> Before	After
> image	image
> https://docs.python.org/3.12/library/sys.html#sys.float_info	https://deploy-preview-101319--python-cpython-preview.netlify.app/library/sys.html#sys.float_info
> And when running Sphinx in nit-picky mode (re: #101100), we're getting warnings about all of those:
> 
> ```
> Doc/library/sys.rst:14: WARNING: py:const reference target not found: epsilon
> Doc/library/sys.rst:19: WARNING: py:const reference target not found: dig
> Doc/library/sys.rst:22: WARNING: py:const reference target not found: mant_dig
> Doc/library/sys.rst:27: WARNING: py:const reference target not found: max_exp
> Doc/library/sys.rst:30: WARNING: py:const reference target not found: max_10_exp
> Doc/library/sys.rst:39: WARNING: py:const reference target not found: min_exp
> Doc/library/sys.rst:42: WARNING: py:const reference target not found: min_10_exp
> Doc/library/sys.rst:45: WARNING: py:const reference target not found: radix
> Doc/library/sys.rst:47: WARNING: py:const reference target not found: rounds
> ```
> 
> So shall we change all of them like this?
> 
> ```diff
> -    :const:`max`
> +    ``max``
> ```
> 
> And we could also do the macros too, although that will also require adjusting the column width:
> 
> ```diff
> -    DBL_MAX
> +    ``DBL_MAX``
> ```

Changing ``` :const:`max` ``` to ``` ``max`` ``` made sense to me. The gray background behind the texts, in my opinion, makes the words stand out more. We can also make these corrections for other words in the documents.

**Review by hugovk — CHANGES_REQUESTED** [review]
Marking as changes requested, re: https://github.com/python/cpython/pull/101319#issuecomment-1405239460

**Review by hugovk — APPROVED** [review]
Looks good now, and the warnings fixed 👍 

image

https://deploy-preview-101319--python-cpython-preview.netlify.app/library/sys.html#sys.float_info

Thanks!

File: Doc/library/sys.rst [hunk 1]
```diff
@@ -568,55 +568,55 @@ always available.
...
+   |                     |                     | a normalized float                               |
+   +---------------------+---------------------+--------------------------------------------------+
+   | ``min_10_exp``      | ``DBL_MIN_10_EXP``  | minimum integer *e* such that ``10**e`` is a     |
+   |                     |                     | normalized float                                 |
+   +---------------------+---------------------+--------------------------------------------------+
+   | ``radix``           | ``FLT_RADIX``       | radix of exponent representation                 |
+   +---------------------+---------------------+--------------------------------------------------+
+   | ``rounds``          | ``FLT_ROUNDS``      | integer representing the rounding mode for       |
+   |                     |                     | floating-point arithmetic. This reflects the     |
+   |                     |                     | value of the system FLT_ROUNDS macro at          |
```
Comment (hugovk) [inline Doc/library/sys.rst]:
```suggestion
   |                     |                     | value of the system ``FLT_ROUNDS`` macro at      |
```

File: Doc/library/sys.rst [hunk 2]
```diff
@@ -568,55 +568,55 @@ always available.
...
+   |                     |                     | interpreter startup time:                        |
+   |                     |                     | ``-1`` indeterminable,                           |
+   |                     |                     | ``0`` toward zero,                               |
+   |                     |                     | ``1`` to nearest,                                |
+   |                     |                     | ``2`` toward positive infinity,                  |
+   |                     |                     | ``3`` toward negative infinity                   |
+   |                     |                     |                                                  |
+   |                     |                     | All other values for FLT_ROUNDS characterize     |
```
Comment (hugovk) [inline Doc/library/sys.rst]:
```suggestion
   |                     |                     | All other values for ``FLT_ROUNDS`` characterize |
```

**Review by hugovk — COMMENTED** [review]
Actually, one last little thing :)

**Comment by hugovk** [comment]
Thank you!

**Pull request merged**